pallets/gear-builtin/src/mock.rs

```diff
@@ -206,16 +210,16 @@ impl pallet_proxy::Config for Test {
 
 parameter_types! {
     pub const BlockGasLimit: u64 = 100_000_000_000;
-    pub const OutgoingLimit: u32 = 1024;
-    pub const OutgoingBytesLimit: u32 = 64 * 1024 * 1024;
+    pub const OutgoingLimit: u32 = OUTGOING_LIMIT;
```

Comment:
these constant should be removed from here at all, as well as from pallets config